pelicanconf.py

```diff
@@ -4,7 +4,7 @@
 
 AUTHOR = "''"
 SITENAME = 'Documentation'
-SITEURL = 'http://anima.rolisteam.org'
+SITEURL = 'http://localhost:8000'
```

Comment:
Is there a way to set the site url in debug as http://localhost:8080/ and as http://anima.rolisteam.org/ when debug flag is enable or something ?

Or at least keep two lines: one for prod and another for debug.

Reply:
It's already the case, when you use the make command (like make ssh_upload) the conf file used is publishconf.py which as the correct anima.rolisteam.org SITEURL.

Reply:
Ok cool!